sdk/communication/Azure.Communication.PhoneNumbers/CHANGELOG.md

```diff
@@ -2,13 +2,8 @@
 
 ## 1.1.0-beta.2 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
 ### Other Changes
+- 'Deprecates' (read discourage) the use of PhoneNumbersModelFactory.PhoneNumberCost(double amount, string currencyCode, string billingFrequency)
```

Comment:
Don't update CHANGELOG this way. This PR is not a version bump?

Reply:
It is not, I think.

No problem I can remove the changelog update